### PR TITLE
ui: update cluster-ui version to v22.1.0-prerelease-3

### PR DESCRIPTION
**Pull request by maryliag**
Update cluster-ui package version

Release note: None

### CODE REVIEW

File: pkg/ui/workspaces/cluster-ui/package.json [hunk 1]
```diff
@@ -40,6 +39,7 @@
     "@babel/preset-env": "^7.11.0",
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.8.0",
+    "@bazel/typescript": "4.1.0",
```
Comment (Azhng) [inline pkg/ui/workspaces/cluster-ui/package.json:42]: Is this intentional?

Comment (maryliag) [reply pkg/ui/workspaces/cluster-ui/package.json:42]: yes

Comment (Azhng) [reply pkg/ui/workspaces/cluster-ui/package.json:42]: I see. Though I think we should probably update the commit msg to prefix it with `ui`. Should we also include release note here ?

**Review by Azhng — APPROVED** [review]

**Pull request merged**